### ISSUE TITLE
Add mesh and pointcloud visualization support

### ISSUE THREAD

**Issue by julianschoep** [allegroai/clearml#1006]
## Proposal Summary
Tensorboard has support for meshes, which are critical for those working with 3D data and point-clouds. Scalars and images are currently already picked up by the clearml interface, however meshes are not. My proposal is to add support for mesh visualization.

## Motivation
As stated above, models working on point-clouds or meshes are often hard to debug without visualizing meshes. Being able to visualize meshes in the clearml interface would enable such use, which sees growing interest in the ML domain.

**Comment by AlexandruBurlacu** [comment]
Hey @julianschoep, maybe you can try to use [`Logger.report_scatter3d`](https://clear.ml/docs/latest/docs/guides/reporting/3d_plots_reporting#3d-scatter-plot) before proper point-cloud support is available

**Comment by julianschoep** [comment]
Yes that could work thanks for pointing it out! I mainly need mesh visualization though. For now I'll upload meshes as artifacts and view them in external software.
